docs/io/unified_table_fits.rst

```diff
@@ -0,0 +1,629 @@
+round-trip any :ref:`mixin_columns` that were written to that HDU, using the
+header information to reconstruct them::
+
+    >>> from astropy.io import fits
+    >>> with fits.open(chandra_events) as hdul:
+    ...     t = Table.read(hdul["EVENTS"])
+
+If a column contains unit information, it will have an associated
+`astropy.units` object::
+
+    >>> t["energy"].unit
+    Unit("eV")
+
+It is also possible to get directly a table with columns as
+`~astropy.units.Quantity` objects by using the `~astropy.table.QTable` class::
+
+    >>> from astropy.table import QTable
+    >>> t2 = QTable.read(chandra_events, hdu=1)
+    >>> t2['energy']
+    <Quantity [7782.7305, 5926.725 ] eV>
+
+Writing
+^^^^^^^
+
+To write a table ``t`` to a new file::
+
+    >>> t.write('new_table.fits')
+
+If the file already exists and you want to overwrite it, then set the
+``overwrite`` keyword::
+
+    >>> t.write('existing_table.fits', overwrite=True)
+
+If you want to append a table to an existing file, set the ``append``
+keyword::
+
+    >>> t.write('existing_table.fits', append=True)
+
+
+.. testcleanup::
+
+    >>> import pathlib
+    >>> pathlib.Path.unlink('new_table.fits')
+    >>> pathlib.Path.unlink('existing_table.fits')
+
+Alternatively, you can use the convenience function
+:func:`~astropy.io.fits.table_to_hdu` to create a single
+binary table HDU and insert or append that to an existing
+:class:`~astropy.io.fits.HDUList`.
+
+There is support for writing a table which contains :ref:`mixin_columns` such
+as `~astropy.time.Time` or `~astropy.coordinates.SkyCoord`. This uses FITS
+``COMMENT`` cards to capture additional information needed order to fully
+reconstruct the mixin columns when reading back from FITS. The information is a
+Python `dict` structure which is serialized using YAML.
+
+Keywords
+^^^^^^^^
+
+The FITS keywords associated with an HDU table are represented in the ``meta``
+ordered dictionary attribute of a :ref:`Table <astropy-table>`. After reading
+a table you can view the available keywords in a readable format using:
+
+  >>> for key, value in t.meta.items():
+  ...     print(f'{key} = {value}')
+  EXTNAME = EVENTS
+  HDUNAME = EVENTS
+  TLMIN2 = 0
+  ...
+
+This does not include the "internal" FITS keywords that are required to specify
+the FITS table properties (e.g., ``NAXIS``, ``TTYPE1``). ``HISTORY`` and
+``COMMENT`` keywords are treated specially and are returned as a list of
+
+  >>> t.meta['MY_KEYWD'] = 'my value'
+  >>> t.meta['COMMENT'] = ['First comment', 'Second comment', 'etc']
+  >>> t.write('my_table.fits', overwrite=True)
+
+.. testcleanup::
+   >>> pathlib.Path.unlink('my_table.fits')
+
+The keyword names (e.g., ``MY_KEYWD``) will be automatically capitalized prior
+to writing.
+
+The ``meta`` attribute of the :class:`~astropy.table.Table` class
+is a dictionary and does not fully represent the structure of a
+FITS header (for example, keyword comments are dropped).
+
+.. _fits_astropy_native:
+
+
+TDISPn Keyword
```

Comment:
My tendency would be to put this nearer the end: I think this is a detail that matters, but less so on how time columns, etc., are dealt with.

Reply:
I'm not sure, but I think this is too far out of scope. It is an an existing detail of the Table-FITS documentation that has not changed in this PR, not about the big-picture refactor.